src/uct/cuda/gdr_copy/gdr_copy_iface.c

```diff
@@ -27,6 +26,19 @@ static ucs_config_field_t uct_gdr_copy_iface_config_table[] = {
      ucs_offsetof(uct_gdr_copy_iface_config_t, super),
      UCS_CONFIG_TYPE_TABLE(uct_iface_config_table)},
 
+    {"BW", "0MBs,get_dedicated:250MBs,put_shared:6911MBs",
+     "Estimated p2p memory copy bandwidth", 0,
+     UCS_CONFIG_TYPE_KEY_VALUE(UCS_CONFIG_TYPE_BW,
+        {"get_dedicated", "dedicated get bandwidth",
+         ucs_offsetof(uct_gdr_copy_iface_config_t, get_bw.dedicated)},
+        {"get_shared", "shared get bandwidth",
+         ucs_offsetof(uct_gdr_copy_iface_config_t, get_bw.shared)},
+        {"put_dedicated", "dedicated put bandwidth",
+         ucs_offsetof(uct_gdr_copy_iface_config_t, put_bw.dedicated)},
+        {"put_shared", "shared put bandwidth",
+         ucs_offsetof(uct_gdr_copy_iface_config_t, put_bw.shared)},
```

Comment:
i think it should be just put and get. Whether it is shared or dedicated should be set/decided implicitly in the transport